packages/components/src/navigation/utils.js

```diff
@@ -0,0 +1 @@
+export const DEFAULT_LEVEL = 'root';
```

Comment:
`Non-blocking`:

I've seen a convention in the Gutenberg repo where we consolidate constants and defaults values in a `constants.js` file. Is there a reason we chose the name `utils.js` here?

Ex.
`packages/block-library/src/image/constants.js`
`packages/block-library/src/embed/constants.js`

Reply:
You're totally right!